README.md

```diff
@@ -160,7 +162,7 @@ install `Command Line Tools for Xcode` and `brew` package manager
 
 run the following commands in the Terminal
 ```
-brew install git cmake
+brew install git cmake rake
```

Comment:
I don't have a Mac to verify this. We likely don't need rake in MacOS, correct?
``` suggestion
brew install git cmake
```

Reply:
```
          _
         (/     remi@MacBook-Pro
    .---__--.   OS:        macOS 12.0.1
   /         \  KERNEL:    Darwin 21.1.0
  |         /   UPTIME:    11 days 23:34
  |         \_  PACKAGES:  30
   \         /  SHELL:     zsh
    `._.-._.`   UI:        Aqua


~
❯ /usr/bin/rake --version
rake, version 13.0.1
```

Yes, `ruby` and `rake` are indeed rooted in macOS 🙃

However, this may change soon as Apple announced they're planning to remove interpreters from the base system (which is probably a good move security-wise).

Regarding `brew`, there is **no such formula** as `rake`!

```
Error: No available formula with the name "rake". Did you mean drake or rke?
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

What you probably meant is `ruby` 😅

```
ruby: stable 3.0.2 (bottled), HEAD [keg-only]
Powerful, clean, object-oriented scripting language
https://www.ruby-lang.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/ruby.rb
License: Ruby
==> Dependencies
Build: pkg-config ✔
Required: libyaml ✘, openssl@1.1 ✘, readline ✘
==> Options
--HEAD
	Install HEAD version
==> Caveats
ruby is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

==> Analytics
install: 147,184 (30 days), 404,746 (90 days), 1,292,257 (365 days)
install-on-request: 79,374 (30 days), 234,665 (90 days), 807,909 (365 days)
build-error: 329 (30 days)
```

I don't personally use brew as my primary package manager, but since `ruby` is keg-only, it makes no sense to install it in this case (for now).